dev/lint-r.R

```diff
@@ -28,6 +28,7 @@ if (! library(SparkR, lib.loc = LOCAL_LIB_LOC, logical.return = TRUE)) {
 # NOTE: The CRAN's version is too old to adapt to our rules.
 if ("lintr" %in% row.names(installed.packages())  == FALSE) {
   devtools::install_github("jimhester/lintr@a769c0b")
+  # devtools::install_github("jimhester/lintr@5431140")
```

Comment:
Here, I didn't yet change this. Up to my knowledge, this doesn't actually try to install this though.. just in case.

Reply:
because of `if ("lintr" %in% row.names(installed.packages())  == FALSE) {`
?
it might be the case the Jenkins boxes already have a version installed and this won't update it

Reply:
Yes .. I think I should add few more changes here if we are going to update this package.

Reply:
It looks like the commented is a old code. Maybe a comment to explain it.

Reply:
Oh, just to be clear, I didn't change this yet but just commented this for now ... Actually, `devtools::install_github("jimhester/lintr@5431140")` is new one but I didn't change this yet just in case.

Will add some comments around this when I add few more changes and clean up soon in any event.